index.js

```diff
@@ -10,7 +10,9 @@ import http from 'http';
 export const config = {
   databaseURI:
     process.env.DATABASE_URI || process.env.MONGODB_URI || 'mongodb://localhost:27017/dev',
-  cloud: process.env.CLOUD_CODE_MAIN || __dirname + '/cloud/main.js',
+  cloud: function () {
+    import('./cloud/main.js');
+  },
```

Comment:
This would ignore the env var; could you refactor? I think the path can be set in an OS agnostic way, using the `url` lib, instead of `__dirname` and for example `url.fileURLToPath`.